tests/accessor/accessor_common.h

```diff
@@ -72,21 +170,33 @@ inline std::string get_section_name(const std::string& type_name,
  * @param section_description String with human-readable description of the test
  * @return std::string String with name for section
  */
-template <int DimensionT>
+template <int Dimension>
 inline std::string get_section_name(const std::string& type_name,
                                     const std::string& access_mode_name,
                                     const std::string& section_description) {
-  using namespace sycl_cts::get_cts_string;
-
-  std::string name = "Test ";
-  name += section_description;
-  name += " with parameters: <";
-  name += type_name;
-  name += "><";
-  name += access_mode_name;
-  name += "><";
-  name += std::to_string(DimensionT) + ">";
-  return name;
+  return section_name(section_description)
+      .with("T", type_name)
+      .with("access mode", access_mode_name)
+      .with("dimension", Dimension)
+      .create();
+}
+
+/**
+ * @brief Function helps to get string section name that will contain template
+ * parameters and function arguments
+ *
+ * @tparam Dimension Integer representing dimension
+ * @param type_name String with name of the testing type
+ * @param section_description String with human-readable description of the test
+ * @return std::string String with name for section
+ */
+template <int Dimension>
+inline std::string get_section_name(const std::string& type_name,
+                                    const std::string& section_description) {
+  return section_name(section_description)
+      .with("T", type_name)
+      .with("dimension", Dimension)
+      .create();
```

Comment:
Despite the fact that this overload from #326, after discuss with @yuriykoch we decided that we can merge #326 and this PR in any order and just resolve conflicts later so there is no dependency between PRs.